tests/plot/test_plot.py

```diff
@@ -12,9 +13,23 @@
     _calculate_distance_to_unit_sphere_zones,
     _unit_sphere_surface,
     plot_3d_heatmap,
+    pxContent_vs_Tau,
 )
 
 
+def test_pxContent_vs_Tau():
+    data = pd.DataFrame(
+        {
+            "umi_per_upia": np.random.uniform(1, 10, 100),
+            "tau": np.random.uniform(0.9, 1, 100),
+            "tau_type": np.random.choice(["high", "low", "normal"], 100),
+            "group": np.random.choice(["A", "B"], 100),
+        }
+    )
+    plot = pxContent_vs_Tau(data, group_by="group")
+    assert plot
```

Comment:
Check my message on how we work with asserting plots

Reply:
I couldn't use the same solution as for the plotly plots since matplotlib plots cannot be exported as json files. 

My solution was to use pytest-mpl instead which makes it possible to add a `@pytest.mark.mpl_image_compare` decorator. Here's a snippet from the pytest-mpl docs:
"Instead of comparing to baseline images, you can instead compare against a JSON library of SHA-256 hashes. This has the advantage of not having to check baseline images into the repository with the tests, or download them from a remote source."

So I ran `pytest --mpl-generate-hash-library=<path_to_file.json>` to generate the JSON library and added the path to the library in the decorator function to enable testing.

Running the test like this:
`pytest --mpl --mpl-generate-summary=html tests/plot/test_plot.py`

will check if the plot matches the has library and also generate an html file summarizing the test results.

Reply:
Nice! You can potentially check in those .json in the repo for development. The tests doesn't get packaged in the production package, so it doesn't matter if they are large (or not).